### PR TITLE
Add stackrox nightly and other ocp jobs for powervs/IbmcloudZ

### PR DESCRIPTION
**Pull request by mdafsanhossain**
This PR re-enables nightly jobs for stackrox on powervs and Ibmcloud Z which was disabled as part of https://github.com/openshift/release/pull/43496

- periodic-ci-stackrox-stackrox-master-daily-ibmcloudz-ebpf-qa-e2e-tests (ebpf)
- periodic-ci-stackrox-stackrox-master-daily-powervs-qa-e2e-tests (ebpf)

Also, adds jobs for other OCP versions,

Power (ebpf):
- powervs-4-12-qa-e2e-tests
- powervs-4-14-qa-e2e-tests

s390x:
- ibmcloudz-4-14-qa-e2e-tests (core-bpf)

### CODE REVIEW

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml [hunk 1]
```diff
@@ -201,7 +201,22 @@ tests:
   optional: true
   steps:
     env:
-      COLLECTION_METHOD: core_bpf
+      COLLECTION_METHOD: ebpf
+    workflow: stackrox-automation-flavors-powervs-e2e
+  timeout: 5h0m0s
+- as: powervs-4-14-qa-e2e-tests
```
Comment (mdafsanhossain) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml:207]: @msugakov @gavin-stackrox Is it fine to add separate jobs for different ocp versions?

Comment (gavin-stackrox) [reply ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml:207]: @mdafsanhossain yes

**Review by gavin-stackrox — CHANGES_REQUESTED** [review]

**Comment by mdafsanhossain** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-master-powervs-4-14-qa-e2e-tests

**Comment by mdafsanhossain** [comment]
@gavin-stackrox ok to test?

**Comment by mdafsanhossain** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-master-powervs-4-14-qa-e2e-tests

**Comment by mdafsanhossain** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-master-powervs-4-14-qa-e2e-tests

**Comment by mdafsanhossain** [comment]
/pj-rehearse periodic-ci-stackrox-stackrox-master-daily-ibmcloudz-ebpf-qa-e2e-tests

**Comment by gavin-stackrox** [comment]
/uncc @vjwilson @msugakov 

@mdafsanhossain you will need to set `TEST_SUITE: powervs-qa-e2e-tests` and `TEST_SUITE: ibmcloudz-qa-e2e-tests` so that the stackrox/scripts/ci/jobs scripts can be found.

**Comment by mdafsanhossain** [comment]
/pj-rehearse periodic-ci-stackrox-stackrox-master-daily-powervs-qa-e2e-tests

**Comment by mdafsanhossain** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-master-powervs-4-14-qa-e2e-tests

**Comment by mdafsanhossain** [comment]
/pj-rehearse periodic-ci-stackrox-stackrox-master-daily-ibmcloudz-ebpf-qa-e2e-tests

**Comment by mdafsanhossain** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-master-powervs-4-14-qa-e2e-tests

**Comment by mdafsanhossain** [comment]
/pj-rehearse periodic-ci-stackrox-stackrox-master-daily-ibmcloudz-ebpf-qa-e2e-tests periodic-ci-stackrox-stackrox-master-daily-powervs-qa-e2e-tests

**Comment by mdafsanhossain** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-master-powervs-4-14-qa-e2e-tests pull-ci-stackrox-stackrox-master-ibmcloudz-qa-e2e-tests-ocp-414

**Comment by mdafsanhossain** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-master-powervs-4-14-qa-e2e-tests

**Comment by mdafsanhossain** [comment]
@gavin-stackrox I have made the changes to the names and also added a 4.12 job for power.

**Comment by dale-fu** [comment]
/pj-rehearse periodic-ci-stackrox-stackrox-master-daily-ibmcloudz-ebpf-qa-e2e-tests
/pj-rehearse pull-ci-stackrox-stackrox-master-ibmcloudz-qa-e2e-tests-ocp-414

**Comment by dale-fu** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-master-ibmcloudz-4-14-qa-e2e-tests
/pj-rehearse periodic-ci-stackrox-stackrox-master-daily-ibmcloudz-4-13-ebpf-qa-e2e-tests

**Comment by gavin-stackrox** [comment]
@mdafsanhossain it looks like errors on the openshift/release side. Trying again:
/pj-rehearse max

**Comment by mdafsanhossain** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-master-ibmcloudz-4-14-qa-e2e-tests pull-ci-stackrox-stackrox-master-powervs-4-12-qa-e2e-tests pull-ci-stackrox-stackrox-master-powervs-4-14-qa-e2e-tests

**Comment by mdafsanhossain** [comment]
@gavin-stackrox I have resolved a conflict and triggered the rest of the pending jobs. 
The daily tests results are here,
[daily-power](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/44286/rehearse-44286-periodic-ci-stackrox-stackrox-master-daily-powervs-4-13-ebpf-qa-e2e-tests/1715268012933648384)
[daily-s390x](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/44286/rehearse-44286-periodic-ci-stackrox-stackrox-master-daily-ibmcloudz-4-13-ebpf-qa-e2e-tests/1715268012887511040)

**Comment by mdafsanhossain** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-master-powervs-4-12-qa-e2e-tests

**Comment by mdafsanhossain** [comment]
/pj-rehearse pull-ci-stackrox-stackrox-master-powervs-4-12-qa-e2e-tests

**Pull request merged**